lib/kontena/machine/vagrant/common.rb

```diff
@@ -0,0 +1,37 @@
+require 'fileutils'
+require 'erb'
+require 'open3'
+require 'json'
+
+module Kontena
+  module Machine
+    module Vagrant
+      module Common
+
+        def erb(template, vars)
+          ERB.new(template).result(OpenStruct.new(vars).instance_eval { binding })
+        end
+
+        def run_command(cmd)
+          exit_code = 0
+          Open3.popen3(cmd) do |stdin, stdout, stderr, wait|
```

Comment:
I suggest something along the lines of this, which seems to work fine for me during test:

```ruby
        def run_command(cmd)
          exit $?.exitstatus unless system(cmd)
        end
```

Ideally it would also print out some kind of CLI error, but `Kontena::Cli::Common.exit_with_error(...)` isn't included in the provisioner classes, (and it's broken as a module function: `NameError : undefined local variable or method `pastel' for Kontena::Cli::Common:Module`). Abusing something like `raise Kontena::Errors::StandardError.new($?.exitstatus, cmd)` also works, although it's the only exception handled by `Kontena::Command` as a loggable error.

```
 [error] 1 : vagrant box update
```